src/java/picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.java

```diff
@@ -20,12 +20,42 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = ConvertSequencingArtifactToOxoG.USAGE,
-        usageShort = ConvertSequencingArtifactToOxoG.USAGE,
+        usage = "Converts all sequencing artifact errors including all bait-bias and pre-adapter artifacts to " +
+                "8-oxoguanine artifacts <br /><br />." +
+                "Oxidation of guanine to 8-oxoguanine can occur during library preparation of genomic DNA via a " +
```

Comment:
I think we could get away with just pointing to the OxoG tool for an explanation of the underlying biochemistry, and focus the explanation on what this tool does (because I don't think it does what the other OxoG tool does, despite the copy/pasted description below...) and why one would run it.